prep-html-syntax/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <title>HTML Syntax Exercise</title>
+    <h1><p style="color:grey;">My favorite Star Wars character is Commander Wolffe!</p></h1>
+    <img src="https://lumiere-a.akamaihd.net/v1/images/ACW_IA_77349_2a55f9c8.jpeg?region=53%2C140%2C1464%2C823" alt="Commander Wolffe in his glory" width="600">
+    <p style="color:grey;">He is the leader of the Wolfpack and served under one of the best Jedi's.</p>
+  </body>
```

Comment:
Hi Jimmy, be sure to divide your page into two specific sections. The `<head>` should open and close be fore the `<body>` opens and closes. The visible content of the page should go in the `<body>`.

```html
<!DOCTYPE html>
<html>
  <head>
    <!-- meta info like <title></title> goes in here -->
  </head>
  <body>
    <!-- visible content like <h1></h1> and <p></p> goes in here -->
  </body>
</html>
```